impl/authorization/authorization_test.go

```diff
@@ -0,0 +1,257 @@
+package authorization
+
+import (
+	"testing"
+
+	"github.com/google/keytransparency/core/authentication"
+
+	authzpb "github.com/google/keytransparency/core/proto/authorization"
+)
+
+const (
+	testUser = "user@example.com"
+	l1       = "r1"
+	l2       = "r2"
+	l3       = "r3"
+	l4       = "r4"
+	l5       = "r5"
+	admin1   = "admin1@example.com"
+	admin2   = "admin2@example.com"
+	admin3   = "admin3@example.com"
+	admin4   = "admin4@example.com"
+	res1     = "1|1"
+	res2     = "1|2"
+	res3     = "1|3"
+	res4     = "1|4"
+)
+
+func setup() *authz {
+	roles1 := &authzpb.AuthorizationPolicy_Roles{
```

Comment:
Good call. I didn't try to imagine it in one big proto, but it looks nicer now :)